integration/flags/.snapshots/TestReportFlagsShouldFail-invalid-context-flag

```diff
@@ -37,4 +37,5 @@ General Flags
 
 
 flag error: scan flag error: invalid context argument; supported values: health
+exit status 1
```

Comment:
Ah, you'll probably need to manually remove these `exit status 1` lines from the snapshots. 

It seems they're generated on local test runs but not by the CI - I'm not sure why but if anyone has an idea, please share 🙏

Reply:
I think if you run the integration tests with the binary (like what scripts/run_test does) it will do the right thing. If you do it with the `go run` command, it won’t

Reply:
Ahh the mystery is solved 🤩 Thanks @cfabianski I will try that on my next PR